crates/joystick/src/common.rs

```diff
@@ -0,0 +1,75 @@
+use evdev::KeyCode;
```

Comment:
Potentially an issue here: This means that evdev needs to be present on both Linux and Windows, even if Windows doesn't use it at all, right? If the suggestion at the end of the file is followed, this line can be removed.